sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusProcessorClient.java

```diff
@@ -107,6 +107,18 @@
         this.tracerProvider = processorOptions.getTracerProvider();
     }
 
+    /**
+     * @return receiver options set by user;
+     */
+    ReceiverOptions getReceiverOptions() {
+        ReceiverOptions receiverOptions = null;
+        final ServiceBusReceiverAsyncClient client = asyncClient.get();
+        if (client != null) {
+            receiverOptions = client.getReceiverOptions();
+        }
+        return receiverOptions;
+    }
```

Comment:
Don't think this method is required. This is added just to test if the value is set. Instead, we should test that the maxLockRenewDuration is honored if it is set in the builder.